webpack/assets/javascripts/bundle_datatables.js

```diff
@@ -1,3 +1,6 @@
 import 'datatables.net-bs/css/dataTables.bootstrap.css';
-import 'datatables.net-bs';
+import dataTable from 'datatables.net';
 
+$.fn.DataTable = dataTable;
+
+require('datatables.net-bs');
```

Comment:
why not import?
if this solve the original issue i was facing with dataTable, maybe it makes sense to send this as a different pr regardless.

Reply:
I thought it should be synced because `datatables.net-bs` using the `$.fn.DataTable`.
I have test it again and it works fine with import  :+1: